src/components/Primary-Button/Button.types.ts

```diff
@@ -0,0 +1,17 @@
+import type { ButtonProps as MuiButtonProps } from '@mui/material/Button';
+
+export type ButtonVariant = 'primary' | 'secondary';
+export type ButtonSize = 'small' | 'medium' | 'large';
+export type ButtonShape = 'rounded' | 'square' | 'pill';
+
+export interface ButtonProps extends Omit<MuiButtonProps, 'variant' | 'color'> {
+  variant?: ButtonVariant;
+  size?: ButtonSize;
+  shape?: ButtonShape;
+  loading?: boolean;
+  fullWidth?: boolean;
+  disabled?: boolean;
+  children: React.ReactNode;
```

Comment:
ButtonProps references React.ReactNode but file does not import React types explicitly. Depending on TS config (jsx: react-jsx / standalone), this may be fine, but to be explicit and avoid type ambiguity it's preferable to import the type directly.

Suggestion:
- Add: import type { ReactNode } from 'react';
- Then change children: ReactNode;

This keeps types explicit and avoids depending on global React type declarations.